dashboards/elasticsearch/elasticsearch-prometheus.json

```diff
@@ -0,0 +1,390 @@
+                "label": "",
+                "value": 50
+              }
+            ],
+            "timeSeriesQuery": {
+              "prometheusQuery": "avg(elasticsearch_jvm_memory_used_bytes / elasticsearch_jvm_memory_max_bytes * 100)"
+            }
+          },
+          "title": "JVM Memory Avg Usage %"
+        },
+        "width": 2,
+        "xPos": 4,
+        "yPos": 0
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "blankView": {},
+            "timeSeriesQuery": {
+              "prometheusQuery": "sum(elasticsearch_cluster_health_number_of_nodes)"
+            }
+          },
+          "title": "ES Nodes"
+        },
+        "width": 2,
+        "xPos": 6,
+        "yPos": 0
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "blankView": {},
+            "timeSeriesQuery": {
+              "prometheusQuery": "sum(elasticsearch_cluster_health_number_of_data_nodes)"
+            }
+          },
+          "title": "ES Data Nodes"
+        },
+        "width": 2,
+        "xPos": 8,
+        "yPos": 0
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "blankView": {},
+            "timeSeriesQuery": {
+              "prometheusQuery": "sum(elasticsearch_cluster_health_number_of_pending_tasks)"
+            }
+          },
+          "title": "Pending Tasks"
+        },
+        "width": 2,
+        "xPos": 10,
+        "yPos": 0
+      },
+      {
+        "height": 3,
+        "widget": {
+          "timeSeriesTable": {
+            "dataSets": [
+              {
+                "tableDisplayOptions": {},
+                "timeSeriesQuery": {
+                  "prometheusQuery": "elasticsearch_process_open_files_count"
+                }
+              }
+            ],
+            "metricVisualization": "NUMBER"
+          },
+          "title": "Open file descriptors"
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 2
+      },
+      {
+        "height": 4,
+        "widget": {
+          "title": "OS Load Average over 15min",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "elasticsearch_os_load15"
+                }
+              },
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "elasticsearch_os_load5"
+                }
+              },
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "elasticsearch_os_load15"
+                }
+              }
```

Comment:
This looks like a duplicate of the plot above, should the query be different, or should this be omitted altogether?